pkg/cmd/server/apis/config/types.go

```diff
@@ -828,6 +831,15 @@ type DNSConfig struct {
 	AllowRecursiveQueries bool
 }
 
+type WebhookTokenAuthenticators struct {
```

Comment:
type should be singular if it describes a single authenticator

Reply:
Ack